mautic/mautic
# fix:[DPMMA-2626] fix focus view count stats

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [Yes]
| New feature/enhancement? (use the a.x branch)      | [No]
| Deprecations?                          | [No]
| BC breaks? (use the c.x branch)        | [No]
| Automated tests included?              | [No] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Fix issue: In focus item details statistics don't load (views and unique views)
https://github.com/mautic/mautic/issues/13412




#### Steps to test this PR:

1: Open any focus item and expand "details"
2: Views and Unique views should load
